src/v3.rs

```diff
@@ -246,6 +246,20 @@ impl Personalization {
             }
         }
     }
+
+    // Add a BCC field
```

Comment:
Can you make this a complete sentence and use three leading slashes? This way it will shot up on rust docs.